src/components/listBox/ListBoxReducers.ts

```diff
@@ -22,15 +20,10 @@ export const listBoxReducer = (state: IListBoxState = listBoxInitialState, actio
 
     switch (action.type) {
         case ListBoxActions.add:
-            const selected = _.chain(action.payload.items)
-                .where({selected: true})
-                .pluck('value')
-                .value();

```

Comment:
I wonder if this is still needed. Otherwise you can't have a default selected value?